### PR TITLE
Telemetry: groups proto

### PR DESCRIPTION
**Pull request by sepehrfrgh**
This PR adds observation tags to group proto for telemetry purposes.
cc: @jimlambrt @ajayreshc

### CODE REVIEW

File: internal/proto/controller/api/resources/groups/v1/group.proto [hunk 1]
```diff
@@ -14,19 +14,19 @@ option go_package = "github.com/hashicorp/boundary/sdk/pbs/controller/api/resour
...
   // Output only. The ID of the Group.
-  string id = 10; // @gotags: `class:"public"`
+  string id = 10; // @gotags: `class:"public" eventstream:"observation"`
 
   // The ID of the scope of which this Group is a part.
-  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public"`
+  string scope_id = 20 [json_name = "scope_id"]; // @gotags: `class:"public" eventstream:"observation"`
 
   // Output only. Scope information for this Group.
   resources.scopes.v1.ScopeInfo scope = 30;
```
Comment (johanbrandhorst) [inline internal/proto/controller/api/resources/groups/v1/group.proto:32]: What does it mean to add this tag to only some fields in a message?

Comment (sepehrfrgh) [reply internal/proto/controller/api/resources/groups/v1/group.proto:32]: These fields are specifically chosen for the telemetry pipeline and for the product team where they are not customer-specified values

Comment (johanbrandhorst) [reply internal/proto/controller/api/resources/groups/v1/group.proto:32]: Maybe I'm missing something - what happens to the fields that are tagged? What values can the `eventstream` tag take?

Comment (sepehrfrgh) [reply internal/proto/controller/api/resources/groups/v1/group.proto:32]: we read and detect them inside observations_event and we will add them to the observation pipeline. for now just an observation value but we might have another type later down the line.

Comment (johanbrandhorst) [reply internal/proto/controller/api/resources/groups/v1/group.proto:32]: So when reviewing this you just want me to check that the right fields are tagged?

Comment (sepehrfrgh) [reply internal/proto/controller/api/resources/groups/v1/group.proto:32]: Exactly, Thank you 🙏

**Review by johanbrandhorst — APPROVED** [review]

File: internal/proto/controller/api/resources/groups/v1/group.proto (hunk 1 above)
Comment (sepehrfrgh) [reply internal/proto/controller/api/resources/groups/v1/group.proto:32]: and for sure If I messed up in any other parts 😄

**Review by jimlambrt — APPROVED** [review]

**Review by ddebko — APPROVED** [review]

**Pull request merged**